etc/icinga2/scripts/mail-host-notification.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/sh
 #
```

Comment:
I'm not a friend of hardcoding shell paths here. I would leave this to `/usr/bin/env` to determine.

Reply:
You're not incorrect, but using /usr/bin/env is just shifting the problem and I haven't seen a system where

Reply:
I agree that that /bin/sh is not guaranteed, but I haven't seen a system where sh wasn't installed under /bin/sh. On the other hand using /usr/bin/env is just shifting the problem and is just as likely to be installed there as when using /bin/sh.

If you really want to be consistent POSIX states the following[0]:
Applications should note that the standard PATH to the shell cannot be assumed to be either /bin/sh or /usr/bin/sh, and should be determined by interrogation of the PATH returned by getconf PATH, ensuring that the returned pathname is an absolute pathname and not a shell built-in.

[0] https://pubs.opengroup.org/onlinepubs/9699919799/utilities/sh.html#tag_20_117_16

Reply:
Honestly, I don't want to start such a discussion. I had too many in the past.

I've just commented on what I thought, now let's see how others feel about it.